res/values/strings.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <string name="app_name">Adblock Plus</string>
+    <string name="app_name">Adblock Plus for Samsung Internet</string>
+    <string name="launcher_name">Adblock Plus</string>
```

Comment:
Just one more thing, you need to use this launcher_name string in AndroidManifest. Under the `<application>` tag, change the label to `android:label="@string/launcher_name"`